recipes/hatchling/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hatchling" %}
+{% set version = "0.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatchling-{{ version }}.tar.gz
+  sha256: 33816429b78d0f069717ccce53a2ec722c79412c86928caa2e01a93b4eaf31a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - hatchling = hatchling.cli:hatchling
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - editables >=0.2
+    - importlib-metadata
+    - packaging >=20.9
+    - pathspec >=0.9
```

Comment:
This (as well as anything else needed for bootstrapping) needs to also be in `host`.

Reply:
(This meaning pathspec)